Makefile

```
@@ -29,27 +29,27 @@ test: all
 target:
 	mkdir -p $@
 
-src/crustls.h: src/*.rs cbindgen.toml
+src/rustls.h: src/*.rs cbindgen.toml
 	cargo check
 	cbindgen --lang C > $@
 
-target/$(PROFILE)/libcrustls.a: src/*.rs Cargo.toml
+target/$(PROFILE)/librustls.a: src/*.rs Cargo.toml
```

```suggestion
target/$(PROFILE)/librustls_ffi.a: src/*.rs Cargo.toml
```
I think, judging by the build failures on this PR